d2l-icons.md

```diff
@@ -6,45 +6,67 @@ Size: `18px` x `18`px
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/angry.svg?raw=true) | angry || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/folder.svg?raw=true) | folder || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/read.svg?raw=true) | read |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/arrow-collapse.svg?raw=true) | arrow-collapse || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/forms.svg?raw=true) | forms || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/reading.svg?raw=true) | reading |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/arrow-expand.svg?raw=true) | arrow-expand || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/fullscreen.svg?raw=true) | fullscreen || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/redo.svg?raw=true) | redo |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/arrow-toggle-down.svg?raw=true) | arrow-toggle-down || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/gear.svg?raw=true) | gear || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/reflection.svg?raw=true) | reflection |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/arrow-toggle-up.svg?raw=true) | arrow-toggle-up || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/glossary.svg?raw=true) | glossary || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/refresh.svg?raw=true) | refresh |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/assignments.svg?raw=true) | assignments || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/grabber-small.svg?raw=true) | grabber-small || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/release-conditions.svg?raw=true) | release-conditions |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/attach.svg?raw=true) | attach || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/grabber.svg?raw=true) | grabber || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/remove-user.svg?raw=true) | remove-user |
+| ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/attendance.svg?raw=true) | attendance || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/grade.svg?raw=true) | grade || ![](https://cdn.rawgit.com/Brightspace/d2l-icons-ui/master/images/tier1/reorder.svg?raw=true) | reorder |
```

Comment:
Looks like despite the attendance icon already existing, the new one is different - mind adding it? (I'm assuming it's just the new design for it or something) Same for the broken link icon.